### PR TITLE
update experimental features for v5.3

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

1. Added v5.3 experimental features 
2. Added the missing release info for some existing experimental features 

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by djshow832 — COMMENTED** [review]
LGTM for the SQL part.

**Review by djshow832 — APPROVED** [review]

**Review by TomShawn — APPROVED** [review]

File: experimental-features.md [hunk 1]
```diff
@@ -22,14 +22,15 @@ aliases: ['/docs-cn/dev/experimental-features-4.0/','/zh/tidb/dev/experimental-f
 
 ## SQL 功能
 
++ [使用 SQL 接口设置数据放置规则](/information-schema/information-schema-placement-rules.md)。(v5.3 实验特性)
```
Comment (qiancai) [inline experimental-features.md:25]:
```suggestion
+ [使用 SQL 接口设置数据放置规则](/placement-rules-in-sql.md)。(v5.3 实验特性)
```

File: experimental-features.md [hunk 2]
```diff
@@ -42,21 +43,25 @@ aliases: ['/docs-cn/dev/experimental-features-4.0/','/zh/tidb/dev/experimental-f
...
-+ [关闭 Titan 功能](/storage-engine/titan-configuration.md#关闭-titan实验功能)。
-+ [Titan Level Merge 功能](/storage-engine/titan-configuration.md#level-merge实验功能)。
++ [关闭 Titan 功能](/storage-engine/titan-configuration.md#关闭-titan实验功能)。（v4.0 实验特性）
++ [Titan Level Merge 功能](/storage-engine/titan-configuration.md#level-merge实验功能)。（v4.0 实验特性）
 + TiFlash 支持将存储引擎的新数据分布在多个硬盘上，分摊 I/O 压力。（v4.0 实验特性）
 
+## 数据迁移
+
++ [DM 的 OpenAPI 功能](https://docs.pingcap.com/zh/tidb-data-migration/stable/open-api)。 （v5.3 实验特性）
+
```
Comment (qiancai) [inline experimental-features.md:53]:
```suggestion
<!--## 数据迁移

+ [DM 的 OpenAPI 功能](https://docs.pingcap.com/zh/tidb-data-migration/stable/open-api)。 （v5.3 实验特性）
-->
```

File: experimental-features.md [hunk 3]
```diff
@@ -42,21 +43,25 @@ aliases: ['/docs-cn/dev/experimental-features-4.0/','/zh/tidb/dev/experimental-f
...
-+ [关闭 Titan 功能](/storage-engine/titan-configuration.md#关闭-titan实验功能)。
-+ [Titan Level Merge 功能](/storage-engine/titan-configuration.md#level-merge实验功能)。
++ [关闭 Titan 功能](/storage-engine/titan-configuration.md#关闭-titan实验功能)。（v4.0 实验特性）
++ [Titan Level Merge 功能](/storage-engine/titan-configuration.md#level-merge实验功能)。（v4.0 实验特性）
 + TiFlash 支持将存储引擎的新数据分布在多个硬盘上，分摊 I/O 压力。（v4.0 实验特性）
 
+<!--## 数据迁移
+
++ [DM 的 OpenAPI 功能](https://docs.pingcap.com/zh/tidb-data-migration/stable/open-api)。 （v5.3 实验特性）
+-->
```
Comment (qiancai) [inline experimental-features.md:53]:
```suggestion
+ [DM 的 OpenAPI 功能](https://docs.pingcap.com/zh/tidb-data-migration/stable/open-api)。 （v5.3 实验特性）-->
```

File: experimental-features.md [hunk 4]
```diff
@@ -42,21 +43,24 @@ aliases: ['/docs-cn/dev/experimental-features-4.0/','/zh/tidb/dev/experimental-f
...
 
-+ [关闭 Titan 功能](/storage-engine/titan-configuration.md#关闭-titan实验功能)。
-+ [Titan Level Merge 功能](/storage-engine/titan-configuration.md#level-merge实验功能)。
++ [关闭 Titan 功能](/storage-engine/titan-configuration.md#关闭-titan实验功能)。（v4.0 实验特性）
++ [Titan Level Merge 功能](/storage-engine/titan-configuration.md#level-merge实验功能)。（v4.0 实验特性）
 + TiFlash 支持将存储引擎的新数据分布在多个硬盘上，分摊 I/O 压力。（v4.0 实验特性）
 
+<!--## 数据迁移
+
++ [DM 的 OpenAPI 功能](https://docs.pingcap.com/zh/tidb-data-migration/stable/open-api)。 （v5.3 实验特性）-->
```
Comment (qiancai) [inline experimental-features.md:52]:
```suggestion
+ [DM 的 OpenAPI 功能](https://docs.pingcap.com/zh/tidb-data-migration/stable/open-api)。 （v5.3 实验特性）-->

```

**Pull request merged**